lib/node-configurator.js

```diff
@@ -69,91 +71,66 @@ exports.configure = function (config, options) {
             [files]
         ]);
 
-        if (langs) {
+        // Base lang techs
+        engines.forEach(function (engine) {
+            nodeConfig.addTech([engine.tech, engine.options]);
+            // (langs[0] !== null) &&
+            nodeConfig.addTarget(engine.target);
+        });
+
+        // For each lang including `null`-lang (false) and mock-lang (true)
+        (isRealLangs ? langs : [langs ? 'lang' : null]).forEach(function (lang) {
+            var langSuffix = '.' + lang + '.js';
+
+            // Real langs need it:
+            isRealLangs && nodeConfig.addTechs([
+                [keysets, { lang: lang }],
+                [i18n, { lang: lang, exports: { globals: 'force' } }]
+            ]);
+
+            if (lang === 'lang') {
+                // Can we replace this with just merging file?
+                nodeConfig.addTech([writeFile, {
+                    target: '?.empty',
+                    content: ''
+                }]);
+                nodeConfig.addTech([mockI18N, {
```

Comment:
We can remove [source](https://github.com/enb/enb-bem-tmpl-specs/blob/master/lib/techs/mock-lang-js.js#L12) option.